.github/workflows/valid-links.yml

```diff
@@ -14,44 +18,33 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+
       - run: |
           git config --global user.email "github-actions[bot]@users.noreply.github.com"
           git config --global user.name "github-actions[bot]"
           git fetch origin master
           git merge origin/master
-      - name: Setup Node
-        uses: actions/setup-node@v1
+      
+      - name: Install .NET 5.0.x
+        uses: actions/setup-dotnet@v1
         with:
-          node-version: '10'
-      - name: Install markdown-link-check
-        run: npm install -g markdown-link-check
+          dotnet-version: '5.0.x'
+
+      - name: Install MarkdownLinkChecker
+        run: dotnet tool install -g markdownlinkchecker
+
       - name: Verify links
         run: |
           deleted_or_renamed=$(git diff --no-commit-id --name-only --diff-filter DR origin/master | grep  -i .md$ | grep -v -i _sidebar.md | grep -v -i ISSUE_TEMPLATE | wc -l)
 
-          if [ "$GITHUB_REF" = "refs/heads/master" ] || [ $deleted_or_renamed -ne 0 ]
+          if [ "$GITHUB_REF" = "refs/heads/master" ] || [[ $deleted_or_renamed -ne 0 ]]
           then
+              echo "Searching all files..."
               files=$(find . -name \*.md ! -iname _sidebar.md ! -ipath \*/ISSUE_TEMPLATE/\*.md -not -path "./.git/*")
           else
               files=$(git diff --no-commit-id --name-only --diff-filter AM origin/master | grep  -i .md$ | grep -v -i _sidebar.md | grep -v -i ISSUE_TEMPLATE | cat)
```

Comment:
The `_sidebar` exclusion could be removed but I don't fully understand these commands so I'd rather let someone else do that later. Having them there won't break anything fwiw.